deploy/nginx.conf

```
@@ -176,6 +176,17 @@ http {
       location /__canary__ {
         auth_basic off;
       }
+      
+      # Redirect to the Cabinet Office's responsible disclosure policy
+      # https://github.com/alphagov/security.txt
+      
+      location /security.txt {
+        return 301 https://vdp.cabinetoffice.gov.uk/.well-known/security.txt;
```

From [their examples](https://github.com/alphagov/security.txt#securitytxt) we'll want a 302?

Although might be wise to use `try_files` to prefer a local **security.txt** if we add one in future

```suggestion
        return 302 https://vdp.cabinetoffice.gov.uk/.well-known/security.txt;
```